### ISSUE TITLE
AMO (addons.mozilla.org) description for Saving Private Imgur 

### ISSUE THREAD

**Issue by grahamperrin** [entriol/vikipediyikurtarmak#1]
Hi

From <https://addons.mozilla.org/en-GB/firefox/addon/saving-private-imgur/>:

> This extension redirects imgur.com to imgurp.com. Nothing more ;)

http://imgurp.com is blank so I was initially confused, a little suspicious of the extension. 

Eventually I found <https://chrome.google.com/webstore/detail/saving-private-imgur/gjgjgomegegkhghlafemhmnghldobcno> with the missing keyword (here, with emphasis): 

> Redirects **blocked** imgur.com to unblocked imgurp.com url. Nothing more or less ;)

So, two points: 

1. you need the word 'blocked' at AMO, for the description to make sense
2. it will help to add a few words about both (a) the blocks; and (b) why http://imgurp.com is blank. 

If you can include screenshots, even better :-)

Good luck

**Comment by mtunad** [comment]
Yes, imgur.com blocked in Turkey for years. For example when you visit reddit.com in Turkey, you will see lot's of broken image urls. imgurp.com is working fine on image urls but not on home page. I'll add the necessary explanations and resubmit it.

Thank you for your suggestions.
